DFC.ServiceTaxonomy.GraphVisualiser/wwwroot/js/webvowl.app.js

```diff
@@ -6283,6 +6283,18 @@ webvowl.app =
                             d3.select("#typeNode").text(node.type());
                             listNodeArray(d3.select("#individuals"), node.individuals());
 
+                        /*Buttons*/
+                            var urlParams = new URLSearchParams(window.location.search);
+                            graphType = urlParams.get('graph');
+
+                            d3.select("#editNode").classed("hidden", !node.editUrl());
+                            d3.select("#resetNodeFocus").classed("hidden", !node.resetFocusUrl());
+
+                            d3.select("#editNode").append("span").text(node.labelForCurrentLanguage());
+                            d3.select('#editNode').on("click", function () { window.open(node.editUrl(), '_blank'); });
+                            d3.select('#resetNodeFocus').on("click", function () { window.open(node.resetFocusUrl() + "&graph=" + graphType, "_self"); });
```

Comment:
_Might_ be worth switching to `location.href`, although the [reasons ](https://stackoverflow.com/questions/4813879/window-open-target-self-v-window-location-href) might well be out of date now.